Packs/Illumio/IllumioCore/IllumioCore.py

```diff
@@ -0,0 +1,95 @@
+import demistomock as demisto
+import urllib3
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from illumio import PolicyComputeEngine
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+ERRORS = {
```

Comment:
As per demisto's code conventions & standards, we've also been following the same practices for other integrations. We do not create any separate modules for constants or any other purpose. We keep only one "python" file throughout the integration.

Reply:
This is irrelevant to the comment above. Just because we can't create a new module doesn't mean we can't use a better pattern than a hardcoded string-to-string map for error messages.

Reply:
Do you want us to create custom exception class for each error message instead of raising valuerrors?

Reply:
The point here is to avoid a hard-coded dictionary that needs hard-coded lookups everywhere we want to use it. We only need these as custom classes or constants if they are reused - if any of the messages are one-offs (likely the invalid Org ID message will only be called in one place) then we can just move them there.

To answer your question though, yes - error messages that get reused should live in a custom exception class.

Reply:
As of now will directly return the error message that is used only once, & for the reusable ones we will create custom exception classes.